docs/api/native-theme.md

```diff
@@ -0,0 +1,35 @@
+# nativeTheme
+
+> Read and respond to changes in Chromium's native color theme
```

Comment:
`nativeTheme` lines up with our `nativeImage` API hence the naming.  IMO it is worth highlighting that this is based off the the theme that the internal Chromium APIs use rather than the OS theme as although they are mostly the same their are differences that could come into play (E.g. the `--force-dark-mode` flag)

Reply:
I'm ok with this - although i still think `appearance` is a more intuitive name. (Apache vote: -0).